extensions/flaticon/src/entities/Token.ts

```diff
@@ -3,16 +3,28 @@ import { LocalStorage } from '@raycast/api';
 type SerializedToken = {
   token: string;
   expires: number;
+  createdAt?: number;
 };
 
 export type Token = {
   token: string;
   expires: number;
+  createdAt?: number;
 };
 
-export const newToken = ({ token, expires }: SerializedToken): Token => ({ token, expires });
-export const emptyToken = () => ({ token: '', expires: 0 });
-export const tokenValid = ({ expires }: Token): boolean => expires - Date.now() / 1000 > 30;
+export const newToken = ({ token, expires }: SerializedToken): Token => ({
+  token,
+  expires,
+  createdAt: Date.now() / 1000,
+});
+export const emptyToken = (): Token => ({ token: '', expires: 0, createdAt: Date.now() / 1000 });
```

Comment:
`0` for the default value maybe would work better: it would show that there is no special meaning there for the empty token

Reply:
Updated!